src/components/common/Metadata.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { shape, string } from "prop-types";
+
+const Metadata = ({ fallbackTitle, metadata, canonicalUrl }) => {
+  if (!metadata) {
+    return <Helmet title={fallbackTitle.substring(0, 60)} />;
+  }
+  const meta = [];
+  metadata.description &&
+    meta.push({ name: "description", content: metadata.description });
+  metadata.keywords &&
+    meta.push({ name: "keywords", content: metadata.keywords });
+  return (
```

Comment:
As a suggestion, you can add some newlines and destructuring here to help readability
```suggestion
  }
  
  const { description, keywords } = metadata;
  let meta = [];
  description && meta = [{ name: "description", content: description }]
  keywords && meta = [...meta, { name: "keywords", content: keywords }]
  
  return (
```

Reply:
thanks ! I change it